lib/ocpp/v201/charge_point.cpp

```diff
@@ -1508,9 +1508,12 @@ AuthorizeResponse ChargePoint::authorize_req(const IdToken id_token, const std::
     // when CSMS does not set cacheExpiryDateTime and config variable for AuthCacheLifeTime is present use the
     // configured AuthCacheLifeTime
     response = call_result.msg;
-    response.idTokenInfo.cacheExpiryDateTime = DateTime(
-        date::utc_clock::now() +
-        std::chrono::seconds(this->device_model->get_value<int>(ControllerComponentVariables::AuthCacheLifeTime)));
+    auto auth_cache_life_time =
+        this->device_model->get_optional_value<int>(ControllerComponentVariables::AuthCacheLifeTime);
+    if (auth_cache_life_time.has_value()) {
+        response.idTokenInfo.cacheExpiryDateTime =
+            DateTime(date::utc_clock::now() + std::chrono::seconds(auth_cache_life_time.value()));
+    }
```

Comment:
```suggestion
```

This should be fixed with: https://github.com/EVerest/libocpp/commit/463975f05a91af41e4b09934bda85f6507db368b